common/src/test/scala/org/broadinstitute/monster/etl/MsgTransformationsSpec.scala

```diff
@@ -24,7 +24,20 @@ class MsgTransformationsSpec extends FlatSpec with Matchers {
   }
 
   it should "continue if a field-to-rename doesn't exist in a message" in {
-    ???
+    val input = Obj(
+      Str("foo") -> Str("bar"),
+      Str("foobar") -> Int32(123),
+      Str("baz") -> Arr(Str("qux"), Float64(1.23))
```

Comment:
Would either:
1. Add one of `abc` or `lol` to this input, to check that partial renames work, or
2. Make a separate test case for the partial-match situation, and change the string description of this test from `if a a field-to-rename doesn't exist` to `if none of the fields-to-rename exist`